addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -874,11 +874,23 @@ registry.colorpicker = SnippetOption.extend({
         color_leave: '_onColorLeft',
         color_reset: '_onColorReset',
     },
+    events: {
+        'click we-select, we-button': '_onToggleColorPalette',
```

Comment:
This and all the related code should not be necessary. There is already code which handle we-select openings, why would we need custom code for this one ?